_tests/phoenix.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+////	Glob Recursive Function
+// Glob Recursively to a Pattern
+function glob_recursive($Pattern, $Flags = 0, $Strip_Underscore = false) {
+	global $settings;
+	$Return = array();
+	// Search in the Current Directory
+	foreach ( glob($Pattern, $Flags) as $File) {
+		if (
+			!$Strip_Underscore ||
+			strpos($File, '/_') === false
+		) {
+			$Return[] = realpath($File);
+		}
+	}
+	// FOREACHDIRECTORY
+	// Search in ALL sub-directories.
+	foreach ( glob(dirname($Pattern).'/*', GLOB_ONLYDIR | GLOB_NOSORT) as $Directory ) {
+		// This is a recursive function.
+		// Usually, THIS IS VERY BAD.
+		// For searching recursively however,
+		// it does make some sense.
+		if (
+			!$Strip_Underscore ||
+			strpos($Directory, '/_') === false
+		) {
+			$Return = array_merge($Return, glob_recursive($Directory.'/'.basename($Pattern), $Flags, $Strip_Underscore));
+		}
+	} // FOREACHDIRECTORY
+	return $Return;
+}
+
+function require_all_once($Directory) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The parameter $Directory is not named in camelCase.](https://www.codacy.com/app/lewisgoddard/phoenix/file/1807499961/issues/source?bid=2030171&fileBranchId=3071992#l34)